stdlib/public/core/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ split_embedded_sources(
     NORMAL BridgeStorage.swift
     NORMAL BridgingBuffer.swift
   EMBEDDED Builtin.swift
-    NORMAL BuiltinMath.swift
+  EMBEDDED BuiltinMath.swift
```

Comment:
This brings in BuiltinMath for all embedded? is that ok? (id say sure... I guess...)

Reply:
I feel like we should eventually conditionalize these + float on something like `hasFeature(HardFloat)` || hasFeature(SoftFloat)`. @stephentyrone do you have any thoughts on this?